### DISCUSSION TITLE
Indexing on Mac with Ollama stalls and shows 404

### DISCUSSION THREAD

**Discussion by chiangs** [logancyang/obsidian-copilot#470] [Q&A] [answered]
Indexing the same vault on Windows works just fine.

When I try to index the vault on Mac, I get a 200 on 'api/chat' but repeated 404s on `api/embeddings' after that.

I am using `ollama` with `Llama3` and the recommended `ollama-nomic-embed-text` embedding model.

<img width="705" alt="Screenshot 2024-06-18 at 12 13 38" src="https://github.com/logancyang/obsidian-copilot/assets/10984593/ad5c685d-a484-426f-8b49-db22573900d7">

**Comment by rajatyofficial** [comment]
i have encountered same error on windows. Getting repeated 404s "/api/embeddings"
![Screenshot 2024-06-19 133545](https://github.com/logancyang/obsidian-copilot/assets/153350825/fea034c4-29bc-4cea-9ec8-0cd490f629b2)

**Comment by chiangs** [comment, marked as answer]
I have solved this. I re pulled `nomic-embed-text` and index everything with no issues. This is also probably your issue @8BitButter since `/api/embeddings` is an endpoint for Ollama running Nomic and nothing really to do with Obsidian/CoPilot for Obsidian.

```
ollama pull nomic-embed-text
```

**Comment by gpfriend** [comment]
When I tell Terminal 
OLLAMA_ORIGINS=app://obsidian.md* ollama serve
I get this:
zsh: command not found: ollama

Suggestions?

**Reply by chiangs** [reply]
you need to install ollama
